app/models/concerns/developers/notifications.rb

```diff
@@ -24,6 +24,10 @@ def notify_as_stale
       send_stale_profile_notification
     end
 
+    def notify_product_feature_update
```

Comment:
This feels a little verbose. What about `send_product_announcment`? Then we can rename the notification `ProductAnnouncementNotification`, too.

Reply:
Should we instead keep it as notify_product_announcement ? so that naming convention is in sync with other methods defined in class like notify_as_stale, invisiblize_and_notify or let me know if send_product_announcement is still fine?